modules/47_regipol/regiCarbonPrice/sets.gms

```diff
@@ -20,6 +20,7 @@ emi_type_47 "emission type used in regional target"
 $ifThen.emiMkt not "%cm_emiMktTarget%" == "off" 
   regiEmiMktTarget(ext_regi)               "regions with emiMkt targets" / /
   regiANDperiodEmiMktTarget_47(ttot,ext_regi) "regions and periods with emiMkt targets" / /
+  regiEmiMktTarget2regi_47(ext_regi,all_regi)  "regions controlled by emiMkt market set to ext_regi" / / 
```

Comment:
sorry, maybe my mind is slow today. 
Does this simply spell out the regions that are in ext_regi for which EmiMktTarget applies?
so if an EmiMktTarget applies to EUR, then this mapping will contain the entries "(EUR,DEU), (EUR,FRA), (EUR, ENC), ..."?
and if you have separate EmiMktTargets for EUR and DEU, it would contain the entries "(DEU,DEU), (EUR,DEU), (EUR,FRA), (EUR, ENC), ..."?

If that is the case, then maybe describe it as "mapping from all ext_regi for which EmiMktTargets apply to their sub-regions"?

Reply:
addressed at https://github.com/remindmodel/remind/pull/1315/commits/cb7953ffda7c42ac9699f3aeb77b95f0f790ede3